### PR TITLE
Document SVGMaskElement.* members

### PR DESCRIPTION
**Pull request by teoli2003**
### Description

Document the members of `SVGMaskElement`

### Motivation

This part of openwebdocs/project#152

### Additional details

To prevent the creation of new flaws:
- I documented `SVGAnimatedLength.animVal`
- I documented `SVGAnimatedLength.baseVal`
- I updated `SVGAnimatedLength` to match the modern structure

### Related issues and pull requests

There are 2 times 2 flaws left, but they will be fixed when #25340 is merged.

### CODE REVIEW

File: files/en-us/web/api/svganimatedlength/animval/index.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+---
+title: SVGAnimatedLength.animVal
+slug: Web/API/SVGAnimatedLength/animVal
+page-type: web-api-instance-property
+browser-compat: api.SVGAnimatedLength.animVal
+---
+
+{{APIRef("SVG")}}
+
+The **`animVal`** property of the {{domxref("SVGAnimatedLength")}} interface contains the current value of a SVG enumeration. If there is no animation, it is the value as the {{domxref("SVGAnimatedLength.baseVal", "baseVal")}}.
```
Comment (dawei-wang) [inline files/en-us/web/api/svganimatedlength/animval/index.md:10]:
```suggestion
The **`animVal`** property of the {{domxref("SVGAnimatedLength")}} interface contains the current value of a SVG enumeration. If there is no animation, it is the same value as the {{domxref("SVGAnimatedLength.baseVal", "baseVal")}}.
```

**Review by dawei-wang — COMMENTED** [review]
add missing word

File: files/en-us/web/api/svganimatedlength/animval/index.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
+---
+title: SVGAnimatedLength.animVal
+slug: Web/API/SVGAnimatedLength/animVal
+page-type: web-api-instance-property
+browser-compat: api.SVGAnimatedLength.animVal
+---
+
+{{APIRef("SVG")}}
+
+The **`animVal`** property of the {{domxref("SVGAnimatedLength")}} interface contains the current value of a SVG enumeration. If there is no animation, it is the same value as the {{domxref("SVGAnimatedLength.baseVal", "baseVal")}}.
```
Comment (sideshowbarker) [inline files/en-us/web/api/svganimatedlength/animval/index.md:10]:
```suggestion
The **`animVal`** property of the {{domxref("SVGAnimatedLength")}} interface contains the current value of an SVG enumeration. If there is no animation, it is the same value as the {{domxref("SVGAnimatedLength.baseVal", "baseVal")}}.
```

File: files/en-us/web/api/svganimatedlength/baseval/index.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
+---
+title: SVGAnimatedLength.baseVal
+slug: Web/API/SVGAnimatedLength/baseVal
+page-type: web-api-instance-property
+browser-compat: api.SVGAnimatedEnumeration.baseVal
+---
+
+{{APIRef("SVG")}}
+
+The **`baseVal`** property of the {{domxref("SVGAnimatedLength")}} interface contains the initial value of a SVG enumeration.
```
Comment (sideshowbarker) [inline files/en-us/web/api/svganimatedlength/baseval/index.md:10]:
```suggestion
The **`baseVal`** property of the {{domxref("SVGAnimatedLength")}} interface contains the initial value of an SVG enumeration.
```

File: files/en-us/web/api/svgmaskelement/maskcontentunits/index.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+The read-only **`maskContentUnits`** property of the {{domxref("SVGMaskElement")}} interface reflects the {{SVGAttr("maskContentUnits")}} attribute. It indicates which coordinate system to use for the contents of the {{SVGElement("mask")}} element.
+
+> **Note:** Although this property is read-only, it is merely a container for two values you can modify, {{domxref("SVGAnimatedEnumeration.baseVal", "baseVal")}} and {{domxref("SVGAnimatedEnumeration.animVal", "animVal")}}.
+
+## Value
+
+An {{domxref("SVGAnimatedEnumeration")}} representing the coordinate system. The possible values are defined in the {{domxref("SVGUnitTypes")}} interface:
+
+- `0` (`SVG_UNIT_TYPE_UNKWOWN`)
+  - : The type is not one of the predefined type.
```
Comment (sideshowbarker) [inline files/en-us/web/api/svgmaskelement/maskcontentunits/index.md:19]:
```suggestion
  - : The type is not one of the predefined types.
```

File: files/en-us/web/api/svgmaskelement/maskcontentunits/index.md [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+- `1` (`SVG_UNIT_TYPE_USERSPACEONUSE`)
+  - : Corresponding to a value of `userSpaceOnUse` for the {{SVGAttr("maskContentUnits")}} attribute and means that all coordinates inside the element refer to the user coordinate system as defined when the mask was created. It is the default value.
```
Comment (sideshowbarker) [inline files/en-us/web/api/svgmaskelement/maskcontentunits/index.md:21]:
```suggestion
  - : Corresponds to a value of `userSpaceOnUse` for the {{SVGAttr("maskContentUnits")}} attribute and means that all coordinates inside the element refer to the user coordinate system as defined when the mask was created. It is the default value.
```

File: files/en-us/web/api/svgmaskelement/maskcontentunits/index.md [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+- `2` (`SVG_UNIT_TYPE_OBJECTBOUNDINGBOX`)
+  - : Corresponding to a value of `objectBoundingBox` for the attribute and means that all coordinates inside the element are relative to the bounding box of the element the mask is applied to. It means that the origin of the coordinate system is the top left corner of the object bounding box and the width and height of the object bounding box are considered to have a length of 1 unit value.
```
Comment (sideshowbarker) [inline files/en-us/web/api/svgmaskelement/maskcontentunits/index.md:23]:
```suggestion
  - : Corresponds to a value of `objectBoundingBox` for the attribute and means that all coordinates inside the element are relative to the bounding box of the element the mask is applied to. It means that the origin of the coordinate system is the top left corner of the object bounding box and the width and height of the object bounding box are considered to have a length of 1 unit value.
```

File: files/en-us/web/api/svgmaskelement/maskunits/index.md [hunk 7]
```diff
@@ -0,0 +1,116 @@
...
+The read-only **`maskUnits`** property of the {{domxref("SVGMaskElement")}} interface reflects the {{SVGAttr("maskUnits")}} attribute of a {{SVGElement("mask")}} element which defines the coordinate system to use for the mask of the element.
+
+> **Note:** Although this property is read-only, it is merely a container for two values you can modify, {{domxref("SVGAnimatedEnumeration.baseVal", "baseVal")}} and {{domxref("SVGAnimatedEnumeration.animVal", "animVal")}}.
+
+## Value
+
+An {{domxref("SVGAnimatedEnumeration")}} representing the coordinate system. The possible values are defined in the {{domxref("SVGUnitTypes")}} interface:
+
+- `0` (`SVG_UNIT_TYPE_UNKWOWN`)
+  - : The type is not one of the predefined type.
```
Comment (sideshowbarker) [inline files/en-us/web/api/svgmaskelement/maskunits/index.md:19]:
```suggestion
  - : The type is not one of the predefined types.
```

File: files/en-us/web/api/svgmaskelement/maskunits/index.md [hunk 8]
```diff
@@ -0,0 +1,116 @@
...
+- `1` (`SVG_UNIT_TYPE_USERSPACEONUSE`)
+  - : Corresponding to a value of `userSpaceOnUse` for the {{SVGAttr("maskUnits")}} attribute and means that all coordinates inside the element refer to the user coordinate system as defined when the mask was created. It is the default value.
+- `2` (`SVG_UNIT_TYPE_OBJECTBOUNDINGBOX`)
+  - : Corresponding to a value of `objectBoundingBox` for the attribute and means that all coordinates inside the element are relative to the bounding box of the element the mask is applied to. It means that the origin of the coordinate system is the top left corner of the object bounding box and the width and height of the object bounding box are considered to have a length of 1 unit value.
```
Comment (sideshowbarker) [inline files/en-us/web/api/svgmaskelement/maskunits/index.md:23]:
```suggestion
  - : Corresponds to a value of `userSpaceOnUse` for the {{SVGAttr("maskUnits")}} attribute and means that all coordinates inside the element refer to the user coordinate system as defined when the mask was created. It is the default value.
- `2` (`SVG_UNIT_TYPE_OBJECTBOUNDINGBOX`)
  - : Corresponds to a value of `objectBoundingBox` for the attribute and means that all coordinates inside the element are relative to the bounding box of the element the mask is applied to. It means that the origin of the coordinate system is the top left corner of the object bounding box and the width and height of the object bounding box are considered to have a length of 1 unit value.
```

**Review by sideshowbarker — APPROVED** [review]

**Pull request merged**